privacy-policy.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: textpage
+title: Privacy Policy
+---
+
+# Privacy Policy
+
+Last updated January 31, 2024
+
+
+
+This privacy notice for Computer Science Society ('we', 'us', or 'our'), describes how and why we might collect, store, use, process, or share your information when you use our services, such as when you:
+ * Visit our website at [cssbham.com](https://cssbham.com), or any website of ours that links to this privacy notice
+ * Join and use our CSS Discord, or any other application of ours that links to this privacy notice
+ * Engage with us in other related ways, including any sales, marketing, or events
+
+#### Questions or concerns?
+Reading this privacy notice will help you understand your privacy rights and choices. If you do not agree with our policies and practices, please do not use our services or discord. If you still have any questions or concerns, please contact us at [css@guild.bham.ac.uk](mailto:css@guild.bham.ac.uk).
+
+
+
+## Summary Of Key Points
+
+
+This summary provides key points from our privacy notice, but you can find out more details about any of these topics by clicking the link following each key point or by using our [table of contents](#table-of-contents) below to find the section you are looking for.
+
+
+**What personal information do we process?**
+When you visit, use, or navigate our services, we may process personal information depending on how you interact with us and the services, the choices you make, and the products and features you use. [Learn more about personal information you disclose to us](#1-what-information-do-we-collect). 
```

Comment:
This passage is unsuitable for a society - we don't have products and features that we make, and "the choices you make" strikes me as very corpo-speak-y.

Reply:
Agreed, we are a university society and need to show that we are friendly and approachable. Too much corporate speak hinders us.